app/src/main/java/bubbleheads/buy_cook/RecipeList.java

```diff
@@ -4,18 +4,22 @@
 
 public class RecipeList {
 
-    private final ArrayList<Recipe> recipes = new ArrayList<>();
+    private static final RecipeList recipeInstance = new RecipeList();
 
-    public RecipeList() {
-        recipes.add(new Recipe("Cocoa cocktail", "milk,cocoa,water", R.drawable.cocoacocktail, Category.DRINKS));
-        recipes.add(new Recipe("Cookie", "mint", R.drawable.pyrig, Category.COOKIES));
-        recipes.add(new Recipe("Sweets", "berries", R.drawable.kex, Category.COOKIES));
-        recipes.add(new Recipe("Coffee", "chocolade, coffee, sugar", R.drawable.coffee, Category.DRINKS));
-        recipes.add(new Recipe("Something tasty", "sugar,fat", R.drawable.pampukh, Category.LUNCH));
-        recipes.add(new Recipe("Pechenko", "sugar, flour,galaretka", R.drawable.pechenko, Category.COOKIES));
-    }
+    public static RecipeList getRecipeInstance() {return recipeInstance;}
+
```

Comment:
this class should have private constructor. You won't create any instance of this class outside of it. Idea is to have only one instance assigned to "instance" field